### PR TITLE
Update version check to handle all higher protocol versions unsupported and [#67] 2.2 java-it-tests

### PR DESCRIPTION
**Pull request by tolbertam**
As cassandra-2.2 uses protocol version 4 Scassandra needs to be updated to version check for it.  Currently ConnectionHandler checks for protocol v3 and will return a 'Ready' message instead of a UnknownVersion in response to a Startup request with v4.   Updated it to check for v3+.

### CODE REVIEW

**Comment by chbatey** [comment]
By no means feel obliged to but a test of this would be to add #67 i.e. just add a new submodule of the it-test that depends on the 2.2 driver and a little copy pasting :)

**Comment by tolbertam** [comment]
Ah yes, that is a great idea and should be pretty straightforward to do.  I'll do that on this branch if that's cool.

File: java-it-tests/driver22/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+dependencies {
+    compile project(':java-it-tests:common')
+    compile('com.datastax.cassandra:cassandra-driver-core:2.2.0-rc1') {
```
Comment (tolbertam) [inline java-it-tests/driver22/build.gradle]: Opened #81 to change this to 2.2.+ once released.

**Comment by chbatey** [comment]
Looks good

**Comment by tolbertam** [comment]
Thanks, seems everything is running cleanly.  As we get v3/v4 features into scassandra, we'll make sure to also add tests to java-it-tests as well.

**Pull request merged**